README.rst

```diff
@@ -181,6 +181,7 @@ Complete configuration for rules can be supplied using a ``json`` formatted stri
 
 A combination of ``--dry-run`` and ``--diff`` will
 display a summary of proposed fixes, leaving your files unchanged.
+Optionally the ``--diff`` can be used to specify the diff. output format; ``--diff diff`` (default) or ``--diff diff-short``.
```

Comment:
The syntax for other options in the document is `--name=value`, could you please use it here as well for consistency?